README.md

```diff
@@ -137,4 +139,5 @@ The validation options available are:
 case eyesNotOpen
 case faceNotStraight
 case faceNotStable(requiredFrames: Int)
+case environmentLuminosity
```

Comment:
Although I understand the reasoning behind naming this differently because it is an option and not an error, the options are now inconsistent. I think we either need to change this to `environmentTooDark`, or rename the other ones e.g. `eyes`, `faceAngle` and `faceStability`.

Reply:
It makes sense your comment Joe. I will bring it back to the team and discuss the naming conventions to find a way to align them 👍